pkg/asset/manifests/infrastructure.go

```diff
@@ -98,10 +103,21 @@ func (i *Infrastructure) Generate(dependencies asset.Parents) error {
 		}
 	case baremetal.Name:
 		config.Status.PlatformStatus.Type = configv1.BareMetalPlatformType
+		// The MAO expects the Provisioining IP in 1.2.3.4/32 CIDR format
+		// But the install-config ClusterProvisioningIP is just the IP
+		provNetIP, provNet, err := net.ParseCIDR(installConfig.Config.Platform.BareMetal.ProvisioningNetworkCIDR)
+		provIP := strings.Replace(provNet.String(), provNetIP, installConfig.Config.Platform.BareMetal.ClusterProvisioningIP
 		config.Status.PlatformStatus.BareMetal = &configv1.BareMetalPlatformStatus{
-			APIServerInternalIP: installConfig.Config.Platform.BareMetal.APIVIP,
-			NodeDNSIP:           installConfig.Config.Platform.BareMetal.DNSVIP,
-			IngressIP:           installConfig.Config.Platform.BareMetal.IngressVIP,
+			APIServerInternalIP:     installConfig.Config.Platform.BareMetal.APIVIP,
+			NodeDNSIP:               installConfig.Config.Platform.BareMetal.DNSVIP,
+			IngressIP:               installConfig.Config.Platform.BareMetal.IngressVIP,
+			ProvisioningInterface:   installConfig.Config.Platform.BareMetal.ProvisioningInterface,
+			ProvisioningNetworkCIDR: installConfig.Config.Platform.BareMetal.ProvisioningNetworkCIDR,
+			ProvisioningIP:          provIP,
+			ProvisioningDHCPStart:   installConfig.Config.Platform.BareMetal.ProvisioningDHCPStart,
+			ProvisioningDHCPEnd:     installConfig.Config.Platform.BareMetal.ProvisioningDHCPEnd,
```

Comment:
https://github.com/openshift/api/pull/480 currently assumes that the DHCP range is provided as comma separated IP address pair where the 1st IP address represents the start of the range and 2nd IP address represents the last address in the range. The enhancement request (https://github.com/openshift/enhancements/pull/90) has been raised with the same format in mind.

Reply:
Ack we can join that here - there was some open discussion on the API PR regarding the comma-separated format vs using a CIDR to express the DHCP range, I'll assume we're going with the comma-separated approach for now.

Reply:
@sadasu another approach to resolve the API abiguity would be to also adopt explicit start/end parameters I guess, but I'll update this to align with the API PR so we can proceed with testing